client/src/api/block_builder/input_selection/core/mod.rs

```diff
@@ -122,7 +122,7 @@ impl InputSelection {
                         let input = self.available_inputs.swap_remove(index);
 
                         // Selects required input.
-                        self.select_input(input)?
+                        self.select_input(input, false)?
```

Comment:
This can also be wrong, no? Required inputs could also be a governance transition

Reply:
Oh yeah I remember putting false saying I would fix it later

Reply:
And I didn't :trollface:

Reply:
But what can we do at this point ? We don't really know if we need a state or governance transition

Reply:
We could look if it's required for outputs/burning or to unlock other required inputs(or features of them) and otherwise assume state transition :thinking:

Reply:
It's handled in transition_alias_input()